Add documentation for extending `IsGranted` attribute (#21331)
symfony/symfony-docs: security.rst

        // ...
    }

You can also extend the ``IsGranted`` attribute to create meaningful shortcuts:

    // src/Security/Attribute/IsAdmin.php
    // ...

    use Symfony\Component\Security\Http\Attribute\IsGranted;

    class IsAdmin extends IsGranted
    {

>         public function __construct() {

Reviewer: ```suggestion
        public function __construct()
        {
```

        // ...
    }

> You can also extend the ``IsGranted`` attribute to create meaningful shortcuts:

Reviewer 2: ```suggestion
You can also extend the ``IsGranted`` attribute to create meaningful shortcuts::
```

        // ...
    }

Extending ``IsGranted``
.......................

> 

Reviewer: ```suggestion
```

> You can also extend the ``IsGranted`` class to create new attributes:

Reviewer: ```suggestion
You can also extend the ``IsGranted`` attribute to create meaningful shortcuts:
```

    // src/Security/Attribute/IsAdmin.php
    // ...

    use Symfony\Component\Security\Http\Attribute\IsGranted;

    class IsAdmin extends IsGranted
    {
        public function __construct() {
            return parent::__construct('ROLE_ADMIN');
        }
    }

>     .. versionadded:: 7.4

Reviewer: ```suggestion
.. versionadded:: 7.4
```

            The :class:`Symfony\\Component\\Security\\Http\\Attribute\\IsGranted`

>             attribute is extendable since Symfony 7.4.

Reviewer: ```suggestion
    The :class:`Symfony\\Component\\Security\\Http\\Attribute\\IsGranted`
    attribute is extendable since Symfony 7.4.
```
